libopeniscsiusr/sysfs.c

```diff
@@ -485,24 +488,32 @@ int _iscsi_iface_kern_id_of_host_id(struct iscsi_context *ctx,
 	_good(_scandir(ctx, sysfs_iface_path, &namelist, &n), rc, out);
 
 	if (n == 0) {
-		rc = LIBISCSI_ERR_SYSFS_LOOKUP;
+		/* this is OK, and needed for transport drivers like 
```

Comment:
There is is a whitespace in `this is OK, and needed for transport drivers like `